packages/cli/test/unit/commands/pull.test.ts

```diff
@@ -98,10 +98,10 @@ describe('pull', () => {
 
       const config = await fs.readJSON(path.join(cwd, '.vercel/project.json'));
       expect(config).toMatchInlineSnapshot(`
-        Object {
```

Comment:
<!-- Questions comments seek clarification.  -->
**question (non-blocking):** ‏was this a different between versions of jest?

Reply:
I might actually start using snapshots now that the syntax is fixed 😌

Reply:
No kidding!